rules/php/security/openssl-cbc-static-iv-php.yml

```
@@ -0,0 +1,651 @@
+  attacks against encrypted data.
+note: >-
+  [CWE-329] Generation of Predictable IV with CBC Mode.
+  [REFERENCES]
+      - https://csrc.nist.gov/publications/detail/sp/800-38a/final
+ast-grep-essentials: true
+utils:
+  Match_pattern_directly_with_prefix_openssl_encryptpart2:
+    kind: function_call_expression
+    all:
+      - has:
+            kind: name
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)